src/Uno.UI/UI/Xaml/Style/Generic/Generic.xaml

```diff
@@ -2804,12 +2804,9 @@
 	<!-- Default style for Windows.UI.Xaml.Controls.Button -->
 	<Style x:Key="XamlDefaultButton"
 				   TargetType="Button">
-		<Setter Property="Background"
-				Value="{ThemeResource SystemControlBackgroundBaseLowBrush}" />
-		<Setter Property="Foreground"
-				Value="{ThemeResource SystemControlForegroundBaseHighBrush}" />
-		<Setter Property="BorderBrush"
-				Value="{ThemeResource SystemControlForegroundTransparentBrush}" />
+        <Setter Property="Background" Value="{ThemeResource ButtonBackground}" />
+        <Setter Property="Foreground" Value="{ThemeResource ButtonForeground}" />
+        <Setter Property="BorderBrush" Value="{ThemeResource ButtonBorderBrush}" />
```

Comment:
Are the following defined somewhere?

```xml
            <!-- Resources for Windows.UI.Xaml.Controls.Button -->
            <StaticResource x:Key="ButtonBackground" ResourceKey="SystemControlBackgroundBaseLowBrush" />
            <StaticResource x:Key="ButtonBackgroundPointerOver" ResourceKey="SystemControlBackgroundBaseLowBrush" />
            <StaticResource x:Key="ButtonBackgroundPressed" ResourceKey="SystemControlBackgroundBaseMediumLowBrush" />
            <StaticResource x:Key="ButtonBackgroundDisabled" ResourceKey="SystemControlBackgroundBaseLowBrush" />
            <StaticResource x:Key="ButtonForeground" ResourceKey="SystemControlForegroundBaseHighBrush" />
            <StaticResource x:Key="ButtonForegroundPointerOver" ResourceKey="SystemControlHighlightBaseHighBrush" />
            <StaticResource x:Key="ButtonForegroundPressed" ResourceKey="SystemControlHighlightBaseHighBrush" />
            <StaticResource x:Key="ButtonForegroundDisabled" ResourceKey="SystemControlDisabledBaseMediumLowBrush" />
            <StaticResource x:Key="ButtonBorderBrush" ResourceKey="SystemControlForegroundTransparentBrush" />
            <StaticResource x:Key="ButtonBorderBrushPointerOver" ResourceKey="SystemControlHighlightBaseMediumLowBrush" />
            <StaticResource x:Key="ButtonBorderBrushPressed" ResourceKey="SystemControlHighlightTransparentBrush" />
            <StaticResource x:Key="ButtonBorderBrushDisabled" ResourceKey="SystemControlDisabledTransparentBrush" />
```

if not, do we need to define them?

Reply:
I discussed this with Martin. As a general rule, we tend to only move over properties from UWP as necessary. In this case, I've copied only what's necessary for the linked issue, but common sense would indicate that it's more reasonable to copy that entire block for the same reasons. @MartinZikmund should probably weigh in on this.

Reply:
In this case you also need to port the resources the style references, as it is very likely they are also missing from our current styles in Uno.UI. These will probably be in `SystemResources.xaml` and there make sure you get those for all themes (Dark/Default, Light and HighContrast)

Reply:
I found the resources present in SystemResources.xaml and also in fluent v2. As for the properties in the style itself, I've just updated the entirety of the default Button style to match Windows. A lot of the differences were due to limitations that are no longer present. I've tested these changes minimally, but obviously I couldn't test every single line change. @MartinZikmund, could you make sure everything makes sense now?

Reply:
I've adjust the themes updates to avoid unnecessary resources adjustments (they fail some runtime tests), and mention that the Sample needs to be using fluent styles to function properly.

There are also two additional runtime test that validate storyboard and setters override.

Reply:
The change does not fix https://github.com/unoplatform/uno/issues/12485, will look further.

Reply:
Ok so the good news is that the fix works properly, but it's rather the playground and the gallery that are not working well when https://github.com/unoplatform/Uno.Themes/pull/1003 is applied. Let's follow-up in https://github.com/unoplatform/Uno.Themes/issues/1059.